src/style.css

```diff
@@ -0,0 +1,65 @@
+.clock {
+  box-shadow: inset 0 0 0 0.02em #fff;
+  display: inline-block;
+  border-radius: 0.5em;
+  position: relative;
+  font-size: 250px;
+  overflow: hidden;
+  height: 1em;
+  width: 1em;
+}
+  
+.clock__hand {
+  margin-left: -0.5em;
+  margin-top: -0.5em;
+  position: absolute;
+  display: block;
+  height: 1em;
+  width: 1em;
+  left: 50%;
+  top: 50%;
+}
+  
+.clock__hand--minute::after {
+  content: "";
+  background-color: red;
+  margin-bottom: -0.02em;
+  margin-left: -0.01em;
```

Comment:
do not do position corrections with margins like this

Can answer the question "Why -0.02em" after a month?